charts/gardener/gardener-resource-manager/values.yaml

```
@@ -19,6 +19,9 @@ controllers:
     concurrentSyncs: 10
   garbageCollector:
     syncPeriod: ""
+  rootCAController:
+    maxConcurrentWorkers: 1
```

```suggestion
    maxConcurrentWorkers: 0
```

Let's disable it by default here